bootstrap/eksctl/crossplane/helm-provider-config.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
```

Comment:
We'll have to update this to Apache 2.0 as well. We moved from aws-samples to awslabs recently and license changed because of that.